### PR TITLE
MGDOBR-1056: Support the "endpoint" error handling strategy

### PR DESCRIPTION
**Pull request by vpellegrino**
**Task**: https://issues.redhat.com/browse/MGDOBR-1056

**Note**: There was no mockup or specific directive on displaying the Error endpoint URL. So, please consider the proposed layout and the description text.

**Screenshots**:
image
image
image

### CODE REVIEW

**Comment by manstis** [comment]
@vpellegrino I was unable to fetch this locally:
```
git checkout -b vpellegrino-MGDOBR-1056 main
git pull https://github.com/vpellegrino/sandbox-ui.git MGDOBR-1056
Switched to a new branch 'vpellegrino-MGDOBR-1056'
remote: Enumerating objects: 36, done.
remote: Counting objects: 100% (36/36), done.
remote: Compressing objects: 100% (8/8), done.
remote: Total 36 (delta 26), reused 36 (delta 26), pack-reused 0
Unpacking objects: 100% (36/36), 4.60 KiB | 188.00 KiB/s, done.
From https://github.com/vpellegrino/sandbox-ui
 * branch            MGDOBR-1056 -> FETCH_HEAD
hint: You have divergent branches and need to specify how to reconcile them.
hint: You can do so by running one of the following commands sometime before
hint: your next pull:
hint: 
hint:   git config pull.rebase false  # merge (the default strategy)
hint:   git config pull.rebase true   # rebase
hint:   git config pull.ff only       # fast-forward only
hint: 
hint: You can replace "git config" with "git config --global" to set a default
hint: preference for all repositories. You can also pass --rebase, --no-rebase,
hint: or --ff-only on the command line to override the configured default per
hint: invocation.
fatal: Need to specify how to reconcile divergent branches.
```
@kelvah has had to rebase the PR against `main` to fix it.

Am I the only person to experience this on every `sandbox-ui` PR?

If so I can try deleting my local fork and setting it up again in case something's broken.

**Comment by vpellegrino** [comment]
> @vpellegrino I was unable to fetch this locally:
> 
> ```
> git checkout -b vpellegrino-MGDOBR-1056 main
> git pull https://github.com/vpellegrino/sandbox-ui.git MGDOBR-1056
> Switched to a new branch 'vpellegrino-MGDOBR-1056'
> remote: Enumerating objects: 36, done.
> remote: Counting objects: 100% (36/36), done.
> remote: Compressing objects: 100% (8/8), done.
> remote: Total 36 (delta 26), reused 36 (delta 26), pack-reused 0
> Unpacking objects: 100% (36/36), 4.60 KiB | 188.00 KiB/s, done.
> From https://github.com/vpellegrino/sandbox-ui
>  * branch            MGDOBR-1056 -> FETCH_HEAD
> hint: You have divergent branches and need to specify how to reconcile them.
> hint: You can do so by running one of the following commands sometime before
> hint: your next pull:
> hint: 
> hint:   git config pull.rebase false  # merge (the default strategy)
> hint:   git config pull.rebase true   # rebase
> hint:   git config pull.ff only       # fast-forward only
> hint: 
> hint: You can replace "git config" with "git config --global" to set a default
> hint: preference for all repositories. You can also pass --rebase, --no-rebase,
> hint: or --ff-only on the command line to override the configured default per
> hint: invocation.
> fatal: Need to specify how to reconcile divergent branches.
> ```
> 
> @kelvah has had to rebase the PR against `main` to fix it.
> 
> Am I the only person to experience this on every `sandbox-ui` PR?
> 
> If so I can try deleting my local fork and setting it up again in case something's broken.

@kelvah @Srihari1192, could you please check if you have the same issue? Thank you

**Comment by Srihari1192** [comment]
> > @vpellegrino I was unable to fetch this locally:
> > ```
> > git checkout -b vpellegrino-MGDOBR-1056 main
> > git pull https://github.com/vpellegrino/sandbox-ui.git MGDOBR-1056
> > Switched to a new branch 'vpellegrino-MGDOBR-1056'
> > remote: Enumerating objects: 36, done.
> > remote: Counting objects: 100% (36/36), done.
> > remote: Compressing objects: 100% (8/8), done.
> > remote: Total 36 (delta 26), reused 36 (delta 26), pack-reused 0
> > Unpacking objects: 100% (36/36), 4.60 KiB | 188.00 KiB/s, done.
> > From https://github.com/vpellegrino/sandbox-ui
> >  * branch            MGDOBR-1056 -> FETCH_HEAD
> > hint: You have divergent branches and need to specify how to reconcile them.
> > hint: You can do so by running one of the following commands sometime before
> > hint: your next pull:
> > hint: 
> > hint:   git config pull.rebase false  # merge (the default strategy)
> > hint:   git config pull.rebase true   # rebase
> > hint:   git config pull.ff only       # fast-forward only
> > hint: 
> > hint: You can replace "git config" with "git config --global" to set a default
> > hint: preference for all repositories. You can also pass --rebase, --no-rebase,
> > hint: or --ff-only on the command line to override the configured default per
> > hint: invocation.
> > fatal: Need to specify how to reconcile divergent branches.
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > @kelvah has had to rebase the PR against `main` to fix it.
> > Am I the only person to experience this on every `sandbox-ui` PR?
> > If so I can try deleting my local fork and setting it up again in case something's broken.
> 
> @kelvah @Srihari1192, could you please check if you have the same issue? Thank you

I am able to clone successfully 
```
MGDOBR-10561]$ git checkout -b vpellegrino-MGDOBR-1056 main
Switched to a new branch 'vpellegrino-MGDOBR-1056'
[svenkata@hari MGDOBR-10561]$ git status
On branch vpellegrino-MGDOBR-1056
nothing to commit, working tree clean
```

**Comment by kelvah** [comment]
> @kelvah @Srihari1192, could you please check if you have the same issue? Thank you

No issues here.

**Review by manstis — APPROVED** [review]
LGTM :+1:

**Review by kelvah — APPROVED** [review]
LGTM :+1: 

Left a minor observation on EHDetail page structure.

File: src/app/Instance/ErrorHandling/ErrorHandlingDetail.tsx [hunk 1]
```diff
@@ -116,6 +120,32 @@ export const ErrorHandlingDetail = ({
               <DescriptionListDescription data-testid="error_handling_method-value">
                 {errorHandlingMethodLabel}
               </DescriptionListDescription>
+              {isEndpointType(errorHandlingType) && errorHandlingParameters && (
+                <DescriptionListDescription>
+                  <Stack hasGutter>
+                    <StackItem>
```
[the remark this answers is not in the document]
Comment (vpellegrino) [reply src/app/Instance/ErrorHandling/ErrorHandlingDetail.tsx]: @kelvah is this what you were expecting? https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/231/commits/e80372a42317a915f03fd380d1920d898beaa603
image

[the remark this answers is not in the document]
Comment (kelvah) [reply src/app/Instance/ErrorHandling/ErrorHandlingDetail.tsx]: :+1:

**Pull request merged**